api/rs/slint/mcu.md

```diff
@@ -43,6 +43,7 @@ In the snippet above, three features are selected:
    feature changes Slint to use unsafe static for storage. This way, you guarantee to use Slint API only from a single thread, and not from interrupt handlers.
  * `libm`: We select this feature to enable the use of the [libm](https://crates.io/crates/libm) crate to provide traits and functions for floating point arithmetic.
    They're typically provided by the Rust Standard Library (std), but that's not available in bare metal environments.
+ * `renderer-software`: This feature is selected to use Slint's built-in software renderer. You may implement or use a different renderer, but the default software renderer should cover most use cases for an MCU environment. 
```

Comment:
Active voice like in the other bullet points. Remove the "other renderer" bit as for MCU environments there is no alternative (or method of implementing a custom renderer).

```suggestion
 * `renderer-software`: We select this feature to use Slint's built-in software renderer. 
```